conda-forge.yml

```
@@ -1,10 +1,13 @@
 build_platform:
   win_64: linux_64
-test: native_and_emulated
-github:
-  branch_name: main
-  tooling_branch_name: main
 conda_build:
   error_overlinking: false
 conda_forge_output_validation: true
+github:
+  branch_name: main
+  tooling_branch_name: main
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
+  linux_s390x: default
```

I haven't checked if these are supported out of the box, TBH.

@isuruf, would you'd like these to run on Travis CI, or rather have them
```suggestion
  linux_aarch64: azure
  linux_ppc64le: azure
  linux_s390x: azure
```
to avoid the usual issues?

---

Nvm, one build failed already; let's try Azure